firefox_media_tests/playback/test_video_playback.py

```diff
@@ -18,28 +18,4 @@ class TestVideoPlayback(MediaTestCase):
     minute each, and is the test that should be run frequently in automation.
     """
 
-    def test_playback_starts(self):
-        with self.marionette.using_context('content'):
-            for url in self.video_urls:
-                try:
-                    video = VideoPuppeteer(self.marionette, url, timeout=60)
-                    # Second playback_started check in case video._start_time
-                    # is not 0
-                    self.check_playback_starts(video)
-                    video.pause()
-                    src = video.video_src
-                    if not src.startswith('mediasource'):
-                        self.marionette.log('video is not '
-                                            'mediasource: %s' % src,
-                                            level='WARNING')
-                except TimeoutException as e:
-                    raise self.failureException(e)
-
-    def test_video_playback_partial(self):
-        """ First 60 seconds of video play well. """
-        with self.marionette.using_context('content'):
-            for url in self.video_urls:
-                video = VideoPuppeteer(self.marionette, url,
-                                       stall_wait_time=10,
-                                       set_duration=60)
-                self.run_playback(video)
+    # Tests are actually implemented in VideoPlaybackTestsMixin.
```

Comment:
This needs `pass` in the class body. It happens to be ok because it's at the end of a module, but would otherwise lead to `IndentationError`.